DataFormats/CTPPSReco/interface/CTPPSPixelLocalTrackRecoInfo.h

```diff
@@ -0,0 +1,19 @@
+/*
+*
+* This is a part of CTPPS offline software.
+* Author:
+*   Andrea Bellora (andrea.bellora@cern.ch)
+*
+*/
+#ifndef DataFormats_CTPPSReco_CTPPSPixelLocalTrackRecoInfo_H
+#define DataFormats_CTPPSReco_CTPPSPixelLocalTrackRecoInfo_H
+
+  /// Track information byte for bx-shifted runs: 
+  /// reco_info = notShiftedRun    -> Default value for tracks reconstructed in non-bx-shifted ROCs
+  /// reco_info = allShiftedPlanes -> Track reconstructed in a bx-shifted ROC with bx-shifted planes only
+  /// reco_info = noShiftedPlanes  -> Track reconstructed in a bx-shifted ROC with non-bx-shifted planes only
+  /// reco_info = mixedPlanes      -> Track reconstructed in a bx-shifted ROC both with bx-shifted and non-bx-shifted planes
+  /// reco_info = invalid          -> Dummy value. Assigned when reco_info is not computed
+  enum class ReconstructionInfo: unsigned short {notShiftedRun = 0, allShiftedPlanes = 1, noShiftedPlanes = 2, mixedPlanes = 3, invalid = 5};
```

Comment:
this should be specified with PPS or CTPPS to avoid conflicts

Reply:
Based on https://github.com/cms-sw/cmssw/pull/25575/files#r248519532
please drop the base type specifications. Otherwise at this point we'll have to decouple the storage representation from the run time.

Reply:
Ok, I'm changing it